### PR TITLE
:sparkles: add image for text comp #2456

### PR DESCRIPTION
**Pull request by BorghildSelle**

### CODE REVIEW

File: web/lib/queries/common/pageContentFields.ts [hunk 1]
```diff
@@ -578,6 +578,16 @@ _type == "keyNumbers" =>{
       }
     },
   },
+  _type == "imageForText" => {
+    "type": _type,
+    "id": _key,
+    "content": content[]{..., ${markDefs}},
+    "aspectRatio": coalesce(aspectRatio, '10:3'),
```
Comment (padms) [inline web/lib/queries/common/pageContentFields.ts:585]: aspect ratio 10:3 is not in allowed list configured ( 16:9 or full width )

File: web/sections/ImageForText/ImageForText.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const ImageForText = forwardRef<HTMLDivElement, ImageForTextProps>(function ImageForText({ anchor, data }, ref) {
+  const { content, image, aspectRatio } = data
+
+  return (
+    <div ref={ref} id={anchor} className={`${aspectRatio === 'fullWidth' ? 'w-full' : 'px-layout-lg'}`}>
+      <div className="">
+        <Image
+          image={image}
+          maxWidth={2000}
+          {...(aspectRatio === 'fullWidth' && { aspectRatio: Ratios.THREE_TO_TEN })}
```
Comment (padms) [inline web/sections/ImageForText/ImageForText.tsx:20]: 10:3 is not valid for this type

Comment (BorghildSelle) [reply web/sections/ImageForText/ImageForText.tsx:20]: Here when using images to 100vw the best ratio was THREE_TO_TEN, so the ratio fullwidth === 3:10

File: sanityv3/schemas/objects/imageForText.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  h3: true,
+  h4: true,
+  internalLink: false,
+  externalLink: false,
+  lists: false,
+  attachment: false,
+  smallText: false,
+})
+
+export type Factbox = {
```
[the remark this answers is not in the document]
Comment (BorghildSelle) [reply sanityv3/schemas/objects/imageForText.tsx]: Ops

File: web/lib/queries/common/pageContentFields.ts (hunk 1 above)
Comment (BorghildSelle) [reply web/lib/queries/common/pageContentFields.ts]: Nice catch

**Pull request merged**